tests/DCPS/SetQosPartition/subscriber.cpp

```diff
@@ -176,6 +176,9 @@ int ACE_TMAIN (int argc, ACE_TCHAR *argv[])
         ACE_OS::sleep (1);
       }
 
+      // Let reader acknowledge the data.
+      ACE_OS::sleep (3);
```

Comment:
Is there another mechanism we could use here?

Reply:
Long term strategy is to break test into two tests (writer changing partitions, reader changing partitions) and remove all sleeps.  Tracking ticket created.